src/main/java/org/nypl/findawayclientlibrary/AudioService.java

```diff
@@ -43,21 +43,19 @@ public AudioEngine getAudioEngine() {
     return audioEngine;
   }
 
+
   /**
    * AudioEngine needs the sessionKey that comes from the Bibliotheca
    * https://partner.yourcloudlibrary.com/cirrus/library/[libraryId]/GetItemAudioFulfillment endpoint.
    */
   public void initAudioEngine(android.content.Context context) {
-    //try {
+    try {
       AudioEngine.init(context, sessionId, LogLevel.VERBOSE);
       audioEngine = AudioEngine.getInstance();
-    //} catch (AudioEngineException e) {
-      //LogHelper.e(TAG, "Error getting audio engine: " + e.getMessage());
-      //e.printStackTrace();
-    //} catch (Exception e) {
-      //LogHelper.e(TAG, "Error getting audio engine: " + e.getMessage());
-      //e.printStackTrace();
-    //}
+    } catch (Exception e) {
+      LogHelper.e(TAG, e, "Error getting audio engine: ", e.getMessage());
+      //TODO: should never happen.  but if it does, kick user back to book detail screen and apologize
```

Comment:
If this should never happen, I would suggest crashing here instead. Forcing the UI to handle an error that should never occur is putting an obligation on part of the code that shouldn't be concerned with such things.

Reply:
The SimplyE app should not crash when the audiobook part of the app crashes.  So I will gently send the user back to their My Book screen and display a message.